src/itemlistformaction.cpp

```diff
@@ -1079,9 +1078,7 @@ void ItemListFormAction::draw_items()
 			return StflRichText::from_plaintext_string("ERROR");
 		}
 		auto& item = visible_items[line];
-		const auto formatted_line = item2formatted_line(item, width, itemlist_format, datetime_format);
-		// TODO: Propagate usage of StflRichText
-		return StflRichText::from_quoted(formatted_line);
+		return  item2formatted_line(item, width, itemlist_format, datetime_format);
```

Comment:
There are two spaces after the `return`. 8-)

Reply:
Fixed, and found another place in that file with two consecutive spaces